modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/ModelController.java

```diff
@@ -41,29 +45,34 @@
     public Boolean getModel() {
         return true;
     }
+
+    @ApiOperation(value = "Get model info by providing metric key")
     @RequestMapping(value = "/api/model", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Move `/api` up to the class level. Also I would prefer to leave the `produces` element off here--if the client accepts `text/xml` then it would be weird to force a json response. What happens if you remove this? Do we try to return XML? (We don't want to return XML but I'd prefer to 406 instead of returning JSON.)

Reply:
Also if you do what I'm saying, you can use `@GetMapping`.